build.cake

```diff
@@ -402,6 +404,90 @@ Task("PackageConsole")
         });
     });
 
+Task("PackageChocolatey")
+	.Description("Creates chocolate packages of the console runner")
+    .IsDependentOn("CreateImage")
+	.Does(() =>
+	{
+		var currentImageDir = IMAGE_DIR + "NUnit-" + packageVersion + "/";
+		
+		EnsureDirectoryExists(PACKAGE_DIR);
+		
+		// Note: Since cake does not yet support a working directory and separate output directory for chocolatey, the following copying and hacks are needed.
+		
+		// List with the addins (addin name, primary dll, additional files)
+		var addins = new Tuple<string, string, string[]>[] {
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.VSProjectLoader",
+				"vs-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitProjectLoader",
+				"nunit-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitV2ResultWriter",
+				"nunit-v2-result-writer.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitV2Driver",
+				"nunit.v2.driver.dll",
+				new [] {
+					"nunit.core.dll",
+					"nunit.core.interfaces.dll",
+					"nunit.v2.driver.addins"
```

Comment:
It probably shouldn't be there either. What you need to do is have a complete path from the `.addins` file in the engine directory to every extension dll. As you have created that file, it goes to the dlls in a single step. There is no `addins/` entry that would cause `tools/addins` to be scanned, so no `tools/addins/*.addins` file would ever be examined. 

IOW, you make a choice about whether to just reference `addins/` or to reference each assembly in your primary `.addins` file. Once that choice is made, it dictates what else you need to do. At this stage of the game, either choice is good - we can change later if we need to - but you don't want to mix the two approaches. 

Hopefully that explains it better rather than adding to the confusion.